test/sync/src/androidTest/kotlin/io/realm/test/mongodb/shared/AppTests.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.test.mongodb.shared
+
+import io.realm.internal.platform.singleThreadDispatcher
+import io.realm.mongodb.App
+import io.realm.mongodb.AppConfiguration
+import io.realm.mongodb.EmailPassword
+import io.realm.mongodb.User
+import io.realm.mongodb.appConfigurationOf
+import io.realm.mongodb.internal.KtorNetworkTransport
+import kotlinx.coroutines.runBlocking
+import kotlin.test.Test
+
+const val TEST_APP_1 = "testapp1"       // Id for the default test app
+const val BASE_URL = "http://127.0.0.1:9090"
+
+class AppTests {
+
+    @Test
+    fun login() {
+        // Send request directly to the local server to get the actual app ID
+        val applicationId = KtorNetworkTransport(
```

Comment:
In Java all of this setup is hidden behind a `TestApp` class. We should probably copy that and associated `close` method to here. But it is fine to just add TODO about it:

https://github.com/realm/realm-java/blob/master/realm/realm-library/src/syncTestUtils/kotlin/io/realm/TestApp.kt#L34

https://github.com/realm/realm-java/blob/master/realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/AppExt.kt#L29

Reply:
Would probably suit well into https://github.com/realm/realm-kotlin/issues/251. Added a TODO referencing this comment.